tests/platform/Earthfile

```
@@ -0,0 +1,137 @@
+VERSION --new-platform 0.6
+
+FROM alpine:3.15
+WORKDIR /work
+
+test:
+    FROM +run-all
+    ARG nativeUname=$(uname -m)
+    ARG tests="regular/linux/arm/v7 regular/linux/arm64 regular/linux/amd64 
+        native user default
+        override/linux/arm64 override/linux/amd64
+        default_override/linux/arm64 default_override/linux/amd64
+        copy_override/linux/arm64 copy_override/linux/amd64
+        copy_other_set/linux/arm64 copy_other_set/linux/amd64
+        copy/linux/arm64 copy/linux/amd64
+        copy_native1 copy_native2"
+    FOR plat IN $tests
+        RUN /bin/sh -c "set -exo pipefail ; \
+            outdir=./out/$plat ; \
+            test -d \$outdir && \
+            cat \$outdir/uname-m && \
+            echo plat=\$plat && \
```

should this one not be escaped? (since it's `FOR plat IN $tests`)?
```suggestion
            echo plat=$plat && \
```

---

(or otherwise, should the above `outdir=./out/$plat ; \` be `outdir=./out/\$plat ; \`?

---

It actually works both ways in this case (it's just a different shell doing the expansion) :D

I've made this consistent now.